packages/bundler/src/BundlerCollectorTracer.ts

```diff
@@ -145,18 +179,14 @@ export function bundlerCollectorTracer (): BundlerCollectorTracer {
         this.countSlot(opcode === 'SLOAD' ? access.reads : access.writes, slot)
       }
 
-      if (opcode === 'REVERT' || opcode === 'RETURN') {
-        const ofs = parseInt(log.stack.peek(0).toString())
-        const len = parseInt(log.stack.peek(1).toString())
-        this.debug.push(opcode + ' ' + toHex(log.memory.slice(ofs, ofs + len)).slice(0, 100))
-      } else if (opcode === 'KECCAK256') {
+      if (opcode === 'KECCAK256') {
         // collect keccak on 64-byte blocks
         const ofs = parseInt(log.stack.peek(0).toString())
         const len = parseInt(log.stack.peek(1).toString())
         // currently, solidity uses only 2-word (6-byte) for a key. this might change..
         // still, no need to return too much
-        if (len < 512) {
-          // if (len == 64) {
+        if (len > 20 && len < 512) {
```

Comment:
I know the values are obvious here, but still please make them WELL_NAMED_CONSTANTS

Reply:
there are no real values here.. actual known values are exactly 64 chars. I usually give well-defined-names to well-defined-values ...

Reply:
Call them A_LITTLE and A_LOT then 😄